craft_store/endpoints.py

```diff
@@ -32,10 +40,26 @@ class Endpoints:
     whoami: str
     tokens: str
     tokens_exchange: str
+    valid_package_types: Sequence[str]
+
+    def _validate_packages(self, packages: Sequence[Package]) -> None:
+        unknown_packages = [
+            p for p in packages if p.package_type not in self.valid_package_types
+        ]
+        if unknown_packages:
+            unknown_package_types = [p.package_type for p in unknown_packages]
+            raise RuntimeError(
```

Comment:
Maybe something more specific, like ValueError?

Reply:
raising ValueError now, I don't know what I was thinking when I thought RuntimeError was the best fit here